motioneye/uploadservices.py

```diff
@@ -833,6 +834,87 @@ def _request_credentials(self, authorization_key):
         }
 
 
+class Nextcloud(UploadService):
+    NAME = 'nextcloud'
+
+    def __init__(self, camera_id):
+        self._server = None
+        self._port = None
+        self._username = None
+        self._password = None
+        self._location = None
+
+        UploadService.__init__(self, camera_id)
+
+    def _get_base_url(self):
+        scheme = 'http://'
+        if self._port == 443:
+            scheme = 'https://'
+        url = scheme + self._server + ':' + str(self._port) + '/remote.php/dav/files/' + self._username + '/'
```

Comment:
```suggestion
        return f”{scheme}{self._server}:{self._port}/remote.php/dav/files/{self._username}/'
```
Faster and easier to read.  The method is named `_get_base_url()` so it is not necessary to assign the value to `url` before returning it.

Reply:
Is it this special `”` wanted here or `"`?

Reply:
Fixed.